huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -362,9 +400,38 @@ func resourceFgsFunctionV2Create(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if err = createFunctionVersions(fgsClient, urn, d.Get("versions").(*schema.Set)); err != nil {
+		return fmt.Errorf("error creating function versions: %s", err)
+	}
+
 	return resourceFgsFunctionV2Read(d, meta)
 }
 
+func createFunctionVersions(client *golangsdk.ServiceClient, functionUrn string, versionSet *schema.Set) error {
+	for _, v := range versionSet.List() {
+		version := v.(map[string]interface{})
+		versionNum := version["name"].(string) // The version name, also name as the version number.
+		// In the future, the function will support manage multiple versions, and will add the corresponding logic to
+		// create versions based on the related API (Create) in this place.
+		aliasCfg := version["aliases"].([]interface{})
+		if len(aliasCfg) < 1 {
+			continue
+		}
```

Comment:
If not specify `aliases`, `name` will be useless.
How about add a validation to make sure `name` and `aliases` mast be specified together?